make build downloads package over ssh instead of https

Hello! I have been trying to build the module for NGINX using the `make` command. This is in our `Dockerfile` for the official NGINX image in our org. 
However, I noticed that since the release of `v.1.2.0`, the `libddwaf` package is downloaded over `ssh` (as opposed to `https`) and this breaks our build. 
I have tried adding the `ssh` config, it fails, but 
# I do not want us to use the `ssh` method to download (even if we later remove the necessary packages and artefacts before shipping), and
# This method is not always guaranteed to succeed (as you can see below) and is very flaky. 

```0.667 + mkdir -p /root/.ssh
0.671 + ssh-agent
0.676 + eval 'SSH_AUTH_SOCK=/tmp/ssh-XXXXXXHHMhFC/agent.16;' export 'SSH_AUTH_SOCK;' 'SSH_AGENT_PID=17;' export 'SSH_AGENT_PID;' echo Agent pid '17;'
0.676 + SSH_AUTH_SOCK=/tmp/ssh-XXXXXXHHMhFC/agent.16
0.676 + export SSH_AUTH_SOCK
0.676 + SSH_AGENT_PID=17
0.676 + export SSH_AGENT_PID
0.676 + echo Agent pid 17
0.676 + ssh-keygen -t rsa -N  -f /root/.ssh/id_rsa
0.677 Agent pid 17
0.828 + ssh-add /root/.ssh/id_rsa
0.831 Identity added: /root/.ssh/id_rsa (root@buildkitsandbox)
0.832 + ssh-keyscan -t rsa github.com
1.428 # github.com:22 SSH-2.0-babeld-32c28593
2.017 + git clone https://github.com/DataDog/nginx-datadog.git --branch v1.2.0
2.019 Cloning into 'nginx-datadog'...
3.905 Note: switching to 'f05587d7a1ac98c1899cc529d1b82941234295aa'.
3.905
3.905 You are in 'detached HEAD' state. You can look around, make experimental
3.905 changes and commit them, and you can discard any commits you make in this
3.905 state without impacting any branches by switching back to a branch.
3.905
3.905 If you want to create a new branch to retain commits you create, you may
3.905 do so (now or later) by using -c with the switch command. Example:
3.905
3.905   git switch -c <new-branch-name>
3.905
3.905 Or undo this operation with:
3.905
3.905   git switch -
3.905
3.905 Turn off this advice by setting config variable advice.detachedHead to false
3.905
3.934 + cd nginx-datadog
3.934 + cat
3.935 + NGINX_VERSION=1.25.5 make build
3.937 git submodule update --init --recursive
3.947 Submodule 'dd-trace-cpp' (https://github.com/DataDog/dd-trace-cpp.git) registered for path 'dd-trace-cpp'
3.947 Submodule 'libddwaf' (git@github.com:DataDog/libddwaf.git) registered for path 'libddwaf'
3.949 Cloning into '/nginx-datadog/dd-trace-cpp'...
6.088 Cloning into '/nginx-datadog/libddwaf'...
7.840 git@github.com: Permission denied (publickey).
7.842 fatal: Could not read from remote repository.
7.842
7.842 Please make sure you have the correct access rights
7.842 and the repository exists.
7.843 fatal: clone of 'git@github.com:DataDog/libddwaf.git' into submodule path '/nginx-datadog/libddwaf' failed
7.843 Failed to clone 'libddwaf'. Retry scheduled
7.846 Cloning into '/nginx-datadog/libddwaf'...
9.574 git@github.com: Permission denied (publickey).
9.576 fatal: Could not read from remote repository.
9.576
9.576 Please make sure you have the correct access rights
9.576 and the repository exists.
9.578 fatal: clone of 'git@github.com:DataDog/libddwaf.git' into submodule path '/nginx-datadog/libddwaf' failed
9.578 Failed to clone 'libddwaf' a second time, aborting
9.579 make: *** [Makefile:31: dd-trace-cpp/.git] Error 1
9.579 make: *** Deleting file 'dd-trace-cpp/.git'```

Hence, can we please download all necessary packages over `https` which do not require additional flaky configuration?

--- a/.gitmodules
+++ b/.gitmodules
@@ -3,4 +3,4 @@
 	url = ../dd-trace-cpp.git
 [submodule "libddwaf"]
 	path = libddwaf
-	url = git@github.com:DataDog/libddwaf.git
+	url = https://github.com/DataDog/libddwaf.git

--- a/.gitmodules
+++ b/.gitmodules
@@ -3,4 +3,4 @@
 	url = ../dd-trace-cpp.git
 [submodule "libddwaf"]
 	path = libddwaf
-	url = https://github.com/DataDog/libddwaf.git
+	url = ../libddwaf.git

